ImageNet/train/dawn_resnet50_tpu_tensorflow17.json

```diff
@@ -0,0 +1,12 @@
+{
+    "version": "v1.0",
+    "author": "Google Cloud TPU",
+    "authorEmail": "cloud-tpu-dawnbench@googlegroups.com",
+    "framework": "TensorFlow v1.7rc1",
+    "codeURL": "https://github.com/tensorflow/tpu/tree/b189540102d6b5b40b1730d7e5ad5c884bae323c/models/experimental/resnet_bfloat16",
+    "model": "ResNet50",
+    "hardware": "GCP N1-standard-2, Cloud TPU",
```

Comment:
Was the base machine type really n1-standard-2? It seems like 2 CPUs is a little low to handle the preprocessing for the TPU.

Reply:
@sb2nov, interesting. In my experience using TensorFlow with GPUs, I've seen that we usually need 16 or 32 preprocessing threads to sustain the GPU's peak compute bandwidth (even if the input data is preprocessed into `TFRecords`). Is this not true for TPUs?

Reply:
TPUs work a bit differently from the GPU. https://cloud.google.com/tpu/docs/tpus covers a bit more detail as limited work on the host machine we don't really need a large VM. The tensorflow server running on the TPU is part of the 6.50 cost of the TPU.

Reply:
In other words, the n1-standard-2 is primarily responsible for (1) generating the initial TF graph and (2) receiving status updates and printing them out. All the downloading of data from GCS and JPEG decode/crop/etc... happens within the Cloud TPU itself.

Reply:
There is a host machine that is directly attached to the TPU. This is separate from the customer VM that generates the initial TF graph.